eox_nelp/user_profile/api/v1/views.py

```diff
@@ -50,6 +51,19 @@ def update_user_data(request):
     try:
         with transaction.atomic():
             accounts.api.update_account_settings(request.user, request.data)
+
+            # This extra code block is required since the method update_account_settings just
+            # allows to update fields defined in the AccountUserSerializer and the AccountLegacyProfileSerializer
+            # so some fields like first_name and last_name are not editable in the standad implementation.
+            extra_account_user_fields = getattr(settings, "EXTRA_ACCOUNT_USER_FIELDS", [])
+
+            if extra_account_user_fields:
+                for field, value in request.data.items():
+                    if field in extra_account_user_fields:
+                        setattr(request.user, field, value)
```

Comment:
```suggestion
                        if hasattr(request.user, field):
                            setattr(request.user, field, value)
```
What do you think of this if check, in case the setting has a wrong value like:
`EXTRA_ACCOUNT_USER_FIELDS = ["cpu_brand"]`
the object would not be modified with that attribute.

Reply:
The change was included in the line 62 however I made an amend instead of a commit